sympy/stats/crv_types.py

```diff
@@ -1260,6 +1261,13 @@ def _cdf(self, x):
         return Piecewise((uppergamma(a,b/x)/gamma(a), x > 0),
                         (S.Zero, True))
 
+    def sample(self):
+        try :
+            from scipy.stats import invgamma
+        except ImportError:
+            skip("Scipy not found")
```

Comment:
This is not the place for "skip", what is being skipped when a user calls sample(X) in their console? The place for skip is in the test file, where you want to skip a test.